src/main/java/com/dreamsportslabs/guardian/config/tenant/TokenConfig.java

```diff
@@ -17,4 +17,5 @@ public class TokenConfig {
   private String cookiePath;
   private Boolean cookieSecure;
   private Boolean cookieHttpOnly;
+  private Boolean additionalClaimsEnabled;
```

Comment:
if additional claims can be added to idtoken also, then maybe changing the field name would make it clear where the claims will be added. 
Unless it is the same for both, on and off at the same time.